src/utils/activity-object-helper.ts

```diff
@@ -15,12 +16,20 @@ export class ActivityObjectHelper {
     object: ActivityObject | Link | URL | Array<ActivityObject | URL>
   ): string | undefined {
     const lang = navigator.language.substr(0, 2);
-    if (ActivityObjectHelper.hasProperty(object, "name")) {
+    if (ActivityObjectHelper.hasProperty(object, "name") && object.name) {
```

Comment:
hasProperty already looks if name is contained in the object. object.name can also throw an error

Reply:
I want to make sure not only that the property exists in the object but as well that it is not empty. How to do that if not with my code I added?

Reply:
Name could still be undefined. So that the linter has no problem with it, you can cast as an ActivityObject. ActivityObject has the property name for sure
```
if (ActivityObjectHelper.hasProperty(object, "name") && (object as ActivityObject).name)
```

Reply:
But how do I know that it actually is an instance of ActivityObject? Could be as well any other instance of a class with the `name` property, right?